dev_guide/builds.adoc

```diff
@@ -912,3 +912,131 @@ case `*scmsecret*`:
 <1> The URL of private repository is usually in the form
 `git@example.com:<username>/<repository>`.
 ====
+
+
+== Limiting the type of build strategies for builds
+
+
+The build strategies that users are allowed to use when creating `*Builds*` or `*BuildConfigs*`
+can be controlled via
+link:../architecture/additional_concepts/authorization.html[authorization policy].
+Each build strategy is controlled via access to a build subresource:
+
+.Build strategy subresources
+[cols="1,1",options="header"]
+|===
+
+|Strategy |Resource
+
+|Docker
+|builds/docker
+
+|Source-To-Image
+|builds/source
+
+|Custom
+|builds/custom
+
+|===
+
+By default, project administrators (`admin` role) and project editors (`edit` role) are granted
+permission to use all strategies (Docker, Source-to-Image, and Custom).
+
+
+=== Disabling a build strategy globally
+To prevent access to a particular build strategy globally, login as cluster administrator and edit
+each of the default roles:
+
+----
+$ oadm edit clusterrole admin
+$ oadm edit clusterrole edit
+----
+
+For each role, remove the line that corresponds to the resource of the strategy to disable:
```

Comment:
I didn't realize we had 'oc patch' until you asked the question :) I will try to come up with a patch... I think that's better than interactive editing